node/src/blockchain/blockchain_interface.rs

```diff
@@ -640,13 +716,13 @@ mod tests {
             std::env::set_var("SIMPLESERVER_THREADS", "1");
             let (tx, rx) = unbounded();
             let _ = thread::spawn(move || {
-                let bodies_arc = Arc::new(Mutex::new(bodies));
+                let bodies_arc: Arc<Mutex<Vec<Vec<u8>>>> = Arc::new(Mutex::new(bodies));
                 Server::new(move |req, mut rsp| {
                     if req.headers().get("X-Quit").is_some() {
                         panic!("Server stop requested");
                     }
                     tx.send(req).unwrap();
-                    let body = bodies_arc.lock().unwrap().remove(0);
+                    let body: Vec<u8> = bodies_arc.lock().unwrap().remove(0);
```

Comment:
Similarly...
We usually don't do this if the compiler knows the truth and you as a human can read it out of the combination of the function signature and the code in the body. I'm saying that just for your better knowledge. I don't find it serious though and won't request a change.